Makefile

```
@@ -0,0 +1,11 @@
+.PHONY: style test install_local_dev
+
+style:
+	ruff api/ --fix
```

```suggestion
	ruff check --fix ./api
```

Use ruff check for calling the linter properly. (ruff contains linter and formtater.)
And ruff usage as below: (from `ruff -h`)
`Usage: ruff [OPTIONS] <COMMAND>`